sensor/common/compliance/node_inventory_handler_impl.go

```diff
@@ -82,28 +84,63 @@ func (c *nodeInventoryHandlerImpl) run() <-chan *central.MsgFromSensor {
 					c.stopC.SignalWithError(errInputChanClosed)
 					return
 				}
-				// TODO(ROX-12943): Do something with the inventory, e.g., attach NodeID
-				c.sendInventory(toC, inventory)
+				c.handleNodeInventory(toC, inventory)
 			}
 		}
 	}()
 	return toC
 }
 
-func (c *nodeInventoryHandlerImpl) sendInventory(toC chan *central.MsgFromSensor, inventory *storage.NodeInventory) {
-	if inventory == nil {
+func (c *nodeInventoryHandlerImpl) handleNodeInventory(toC chan *central.MsgFromSensor, inventory *storage.NodeInventory) {
+	// TODO(ROX-12943): Replace fakeNode with proper solution for finding the node
+	node := c.fakeNode(inventory)
+	c.sendNode(toC, node)
+}
+
+func (c *nodeInventoryHandlerImpl) sendNode(toC chan *central.MsgFromSensor, node *storage.Node) {
+	if node == nil {
 		return
 	}
 	select {
 	case <-c.stopC.Done():
+		return
 	case toC <- &central.MsgFromSensor{
 		Msg: &central.MsgFromSensor_Event{
 			Event: &central.SensorEvent{
-				Resource: &central.SensorEvent_NodeInventory{
-					NodeInventory: inventory,
+				Id:     node.GetId(),
+				Action: central.ResourceAction_CREATE_RESOURCE, // TODO(ROX-12943): find proper value to use here: create vs. update
+				Resource: &central.SensorEvent_Node{
+					Node: node,
 				},
 			},
 		},
 	}:
 	}
 }
+
+func (c *nodeInventoryHandlerImpl) fakeNode(inventory *storage.NodeInventory) *storage.Node {
```

Comment:
This function is temporary and will be replaced in the follow-up PR. I used it here for manually testing as it allows us to send messages to Central and see the fake node appear in the UI.